controllers/imagejob/imagejob_controller.go

```diff
@@ -197,9 +197,17 @@ func (r *Reconciler) Reconcile(ctx context.Context, req ctrl.Request) (ctrl.Resu
 }
 
 func podListOptions(jobTemplate *corev1.PodTemplate) client.ListOptions {
+	var set map[string]string
+
+	if jobTemplate.Template.Spec.Containers[0].Name == "remover" {
+		set = map[string]string{"type": "manual"}
```

Comment:
We should namespace our label keys under `eraser.sh`, so this should be `eraser.sh/type`. The reason being that if some other extension uses `type` in its logic, we wouldn't want to conflict with that.